docs/content/configuration/index.md

```diff
@@ -248,6 +248,20 @@ You can enable caching of results at the broker/historical using following confi
 |`druid.cache.maxObjectSize`|Maximum object size in bytes for a Memcached object.|52428800 (50 MB)|
 |`druid.cache.memcachedPrefix`|Key prefix for all keys in Memcached.|druid|
 
+#### Redis
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.cache.host`|Redis host|127.0.0.1|
+|`druid.cache.port`|Redis port|6379|
+|`druid.cache.timeout`|Connection / operations timeout|30000|
+|`druid.cache.prefix`|Key prefix|druid|
+|`druid.cache.expiration`|Redis key expiration time|2592000 (30 days)|
```

Comment:
is it possible to have no expiration at all and let redis handle LRU expiration when memory is full?